internal/dataplane/translator/translate_utils.go

```diff
@@ -96,20 +97,43 @@ func generateKongServiceFromBackendRefWithName(
 	// check if the service is already known, and if not create it
 	service, ok := rules.ServiceNameToServices[serviceName]
 	if !ok {
+		backendRequestTimeout := DefaultServiceTimeout
+
 		service = kongstate.Service{
 			Service: kong.Service{
 				Name:           kong.String(serviceName),
 				Host:           kong.String(serviceHost),
 				Protocol:       kong.String(protocol),
-				ConnectTimeout: kong.Int(DefaultServiceTimeout),
-				ReadTimeout:    kong.Int(DefaultServiceTimeout),
-				WriteTimeout:   kong.Int(DefaultServiceTimeout),
+				ConnectTimeout: kong.Int(backendRequestTimeout),
+				ReadTimeout:    kong.Int(backendRequestTimeout),
+				WriteTimeout:   kong.Int(backendRequestTimeout),
 				Retries:        kong.Int(DefaultRetries),
 			},
 			Namespace: route.GetNamespace(),
 			Backends:  backends,
 			Parent:    route,
 		}
+
+		resource, ok := route.(*gatewayapi.HTTPRoute)
+		if ok {
+			for _, rule := range resource.Spec.Rules {
+				if rule.Timeouts != nil && rule.Timeouts.BackendRequest != nil {
+					duration, err := time.ParseDuration(string(*rule.Timeouts.BackendRequest))
+					if err != nil {
+						continue
```

Comment:
I intially wanted to leave a comment to somehow propagate this error up (or at least log it), but I checked in reality it should never happen because it's validated on API server level. Could we leave a comment here explaining that `continue` (e.g.`Timeouts.BackendRequest` is validated to be a strict subset of Golang `time.ParseDuration` so it should never happen)?